lib/shopify-cli/commands/connect.rb

```diff
@@ -53,15 +67,27 @@ def get_shop(shops, id)
         shop
       end
 
-      def write_env(app, shop)
+      def write_env(app, shop, scopes, extra)
+        scopes = 'write_products,write_customers,write_draft_orders' if scopes.nil?
+        extra = {} if extra.nil?
```

Comment:
I started by doing that, but the defaults don't get used if the fields are passed as `nil`. It only uses defaults if you don't pass the fields at all. I found it harder to optionally send params when calling the method since there would be 3 combinations.

Reply:
That's a fair point, so you're covering for nils in any case - makes sense to me.

Kind of related: I suspect there aren't any calls that won't use `scopes` and `extra`, right? If there are, we might consider defaulting them to nil to make calling this method simpler.